src/components/dialogs/parameters/parameters.js

```diff
@@ -581,9 +602,13 @@ const Parameters = ({ user, isParametersOpen, hideParameters }) => {
                                         loadFlowParametersBackend
                                     }
                                     showAdvancedLfParams={showAdvancedLfParams}
+                                    showSpecificLfParams={showSpecificLfParams}
                                     setShowAdvancedLfParams={
                                         setShowAdvancedLfParams
                                     }
+                                    setShowSpecificLfParams={
```

Comment:
I don't get why those states are in parameters.js instead of directly in their components : `AdvancedLoadFlowParameters` and `SpecificLoadFlowParameters` ??

Reply:
fixed in #1239